core/src/main/scala/com/salesforce/op/filters/Summary.scala

```diff
@@ -59,8 +59,8 @@ case object Summary {
    */
   def apply(preppedFeature: ProcessedSeq): Summary = {
     preppedFeature match {
-      case Left(v) => Summary(v.size, v.size, v.size, 1.0)
-      case Right(v) => monoid.sum(v.map(d => Summary(d, d, d, 1.0)))
+      case Left(v) => Summary(v.size, v.size, v.size, 1.0, v.map(d => hllMonoid.create(d.getBytes)).reduce(_+_))
+      case Right(v) => monoid.sum(v.map(d => Summary(d, d, d, 1.0, hllMonoid.create(Array(d.toByte)))))
```

Comment:
`d.toByte` seems invalid to me. Google around on how to convert Double value to array of bytes.

Reply:
Most solutions i found are kinda clunky. How do you feel about `hllMonoid.create(d.toString.getBytes)` ?

Reply:
Either way you do it: a) it has to be fast b) is has to be consistent. I bet there is already some utility in our dependencies that does Double -> Array[Byte] conversion for you.